modules/alloy/src/main/ts/ephox/alloy/positioning/mode/ContentAnchorCommon.ts

```diff
@@ -9,8 +9,11 @@ import * as Origins from '../layout/Origins';
 import { Anchoring, NodeAnchor, nu as NuAnchor, SelectionAnchor } from './Anchoring';
 import * as AnchorLayouts from './AnchorLayouts';
 
-const capRect = (left: number, top: number, width: number, height: number): Optional<Boxes.BoxByPoint> => {
-  let newLeft = left, newTop = top, newWidth = width, newHeight = height;
+const capRect = (left: number, top: number, width: number, height: number, cappingViewport: Optional<Boxes.Bounds>): Optional<Boxes.BoxByPoint> => {
```

Comment:
I think this function could do with an overall comment. It seems to be doing something like:

Adjust the left, top, width, and height so that the rectangle is contained *completely* within the cappingViewport. Although at the moment, it is actually capping to (0, 0, cap.width, cap.height). You probably want to mention "why" it does that also.